src/js/quiz-factory.js

```diff
@@ -10,4 +10,25 @@ const Quiz = (question, choices, answer) => {
   return { getQuestion, getChoices, getAnswer, setPassed, getPassed };
 };
 
-export default Quiz;
+const QuizSummary = (quizArray) => {
+  let passed = false;
+  let quizzes = [];
+
+  for (let i=0; i < quizArray.length; i++) {
+    let quiz = Quiz(quizArray[i].question, quizArray[i].choices, quizArray[i].answer);
+    quizzes.append(quiz);
```

Comment:
Do you mean `quizzes.push(quiz)`?

Reply:
whoops, yeah that's meant to be push lol. ill fix that.